cogs/gamerwords.py

```diff
@@ -122,9 +215,30 @@ async def on_message_edit(self, old_message, new_message):
 		await self.handle_new_gamer_message(new_message)
 
 	async def handle_new_gamer_message(self, message):
-		match = self.has_gamer_words(message.content)
-		if match:
+		text_match = self.has_gamer_words(message.content)
+		file_match = any(
+			[self.has_gamer_words(attachment.filename) for attachment in message.attachments]
```

Comment:
pass a generator to any() instead of a list comp. Also that probably fits in 120 characters so make it one line.